### DISCUSSION TITLE
Eclipse Sleigh Editor installed but how to get it to work?

### DISCUSSION THREAD

**Discussion by hippietrail** [NationalSecurityAgency/ghidra#6961] [Q&A] [answered]
I followed the steps in the GhidraSleighEditor_README to install the Sleigh Editor until this point:

> **Building Eclipse Sleigh Editor**
> To build the Sleigh Editor, follow the instructions in ghidra/DevGuide.md to setup eclipse for development. Then follow the directions in _ghidra/GhidraBuild/EclipsePlugins/GhidraSleighEditor/ghidra.xtext.sleigh.feature/build_README.txt_.
> 
But this file doesn't exist, not even in the GitHub repo:
<img width="875" alt="image" src="https://github.com/user-attachments/assets/f7866e74-88f7-4390-b2aa-015c2891bf5c">

If I ignore this step and just open Sleigh files in Eclipse, they just open as plain text files with no syntax highlighting or any extra features I can see:  
<img width="950" alt="slaspec file open with installed software showing Ghidra Sleigh Editor" src="https://github.com/user-attachments/assets/e376af5c-f28d-4cad-85e3-3dc3635a6b4b">

Am I just missing something obvious or is the documentation out of date, or is it broken?

**Comment by GhidorahRex** [comment, marked as answer]
The documentation is incorrect. The build_README file is in the same `ghidra.xtext.sleigh` directory, not `ghidra.xtext.sleigh.feature`. Sorry for the mixup!
